File: liemipeli/src/main/java/com/antero/liemipeli/domain/Kurssi.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.antero.liemipeli.domain;

/**
 *
 * @author Antero Oikkonen
 */
public class Kurssi {
    private String kurssinNimi;
    private int opintopisteet;
    private int kurssinHinta;
    
    
    
    public String getKurssinNimi() {
        return kurssinNimi;
    }

    public void setKurssinNimi(String kurssinNimi) {
        this.kurssinNimi = kurssinNimi;
    }

    public int getOpintopisteet() {
        return opintopisteet;
    }

    public void setOpintopisteet(int opintopisteet) {
        this.opintopisteet = opintopisteet;
    }

    public int getKurssinHinta() {
        return kurssinHinta;
    }

    public void setKurssinHinta(int kurssinHinta) {
        this.kurssinHinta = kurssinHinta;
    }
    
    
}

File: liemipeli-kotlin/src/main/kotlin/com/antero/liemipeli/opiskelu/Kurssi.kt
package com.antero.liemipeli.opiskelu

/**
 *
 * @author Antero Oikkonen
 */
class Kurssi {
    var kurssinNimi: String? = null
    var opintopisteet: Int = 0
    var kurssinHinta: Int = 0

    override fun toString(): String {
        return "\nKurssin nimi: $kurssinNimi\nkurssin hinta: $kurssinHinta\nopintopisteet $opintopisteet"
    }
}

